src/test/java/info/movito/themoviedbapi/TmdbTrendingTest.java

```diff
@@ -33,7 +33,7 @@ public TmdbTrending createApiToTest() {
     @Test
```

Comment:
Please could you add tests for the new methods added?

I think the existing tests can stay as they were because without a page query parameter added, it will default to page 1.